src/components/common/YourCompany/NameTranslations/AddNameTranslation.vue

```diff
@@ -68,25 +69,24 @@ export default class AddNameTranslation extends Mixins(CommonMixin) {
     nameTranslationForm: FormIF
   }
 
-  @Prop({ default: '' })
-  readonly editNameTranslation: string
+  @Prop({ default: '' }) readonly editNameTranslation!: string
 
-  @Prop({ default: -1 })
-  readonly editNameIndex: number
+  @Prop({ default: -1 }) readonly editNameIndex!: number
 
   // Local Properties
-  protected nameTranslationForm: boolean = false
-  protected nameTranslation: string = ''
-  protected nameIndex: number = -1
+  protected nameTranslationForm = false
+  protected nameTranslation = ''
+  protected nameIndex = -1
```

Comment:
Types are inferred from initial value, and future versions of lint (*) actually warn about "redundant typing".

(*) or maybe it's a lint option enabled in some cases.